distribution/test/itests/pom.xml

```diff
@@ -78,7 +80,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.11</version>
+                <version>${junit.version}</version>
```

Comment:
Since junit and hamcrest are already listed as dependencies in the root pom file, I don't think we need them here in the `<dependencyManagement>` section.